js/todo-modal.js

```diff
@@ -0,0 +1,45 @@
+//open from desktop
+const toDoAppIcon = document.querySelector("#toDo");
+const toDoModal = document.querySelector("#toDoApp");
+
+toDoAppIcon.addEventListener("dblclick", () => {
+  toDoModal.showModal();
+})
+
+const closeToDoButton = document.querySelector("#closeToDo");
+closeToDoButton.addEventListener("click", () => {
+  toDoModal.close();
+})
+
+//open from menu
+document.querySelector("#toDoMenuItem").addEventListener("click", () => {
+  document.querySelector("#menu").style.display = "none";
+  toDoModal.showModal();
+})
+
+//add items to list
+const addTaskButton = document.querySelector("#addTaskBtn");
+const taskList = document.querySelector("#taskList");
+const toDoInput = document.querySelector("#toDoInput");
+
+addTaskButton.addEventListener("click", () => {
+  const newListItem = document.createElement("li");
+  newListItem.classList.add("tasklist-item");
+  newListItem.innerText = toDoInput.value;
+  taskList.appendChild(newListItem);
+  toDoInput.value = "";
+
+  newListItem.addEventListener("click", () => {
+    newListItem.style.textDecoration = "line-through";
+  })
+  newListItem.addEventListener("dblclick", () => {
+    taskList.removeChild(newListItem);
+  })
```

Comment:
I know you have explanation for how it works but I would maybe make it more user friendly, for example a checkbox a user can click to tick off a completed task and maybe an "x" button to remove? this is just a suggestion :)